docs/themes/oso-tailwind/layouts/partials/toc.html

```diff
@@ -1,7 +1,16 @@
 <div class="grid grid-cols-2 md:grid-cols-3 gap-2 justify-center py-4 leading-normal">
   {{ $pages := .Pages }}
+  {{ $filteredPages := slice }}
   {{ if .Params.reverse }}{{ $pages = $pages.Reverse }}{{ end }}
-  {{ $pages = first 6 $pages }}
+
+  {{/* filter out pages w/ `_build: render: never` in the frontmatter */}}
```

Comment:
Can you check we have this for all the pages with `href`s in the params?

Reply:
this = `_build: render: never`

Reply:
yes, confirmed